### ISSUE TITLE
Sync Error 1.0.117

### ISSUE THREAD

**Issue by dersimmy** [laurent22/joplin#1008]
### Operating system
- Linux

### Application
- Desktop

Hi, 

since the last Update I can't sync anymore with my Nextcloud. I get an error message:

 TypeError: Cannot use 'in' operator to search for 'd:multistatus' in null

**Comment by foxmask** [comment]
on my side, on the mobile (android), the sync does nothing. the log do not move at all, the message just displays "X annuler synchornisation" and nothing move at all

**Comment by foxmask** [comment]
@dersimmy Yesterday, I could sync on Linux w/ Dropbox.

**Comment by foxmask** [comment]
for me the android version does not work.
i'm stuck once the sync starts

**Comment by dersimmy** [comment]
Sorry for closing, I misinterpreted your comment. 
For me the android App is working well.

**Comment by foxmask** [comment]
Ah ?
Which version 1.0.176?

**Comment by dersimmy** [comment]
Yes, on my phone is 1.0.176 installed and sync works perfectly using Nextcloud.
